extensions/windows-event-log/tests/custom-provider/generate-and-register.bat

```diff
@@ -16,6 +16,12 @@ rem limitations under the License.
 
 cd %1
 
+set man_path=%1"\unit-test-provider.man"
+set man_path=%man_path:"=%
```

Comment:
A comment about what this does could be useful here, as I suspect I am not the only one who doesn't know the syntax of Windows variable substitution, and this is difficult to google.

Reply:
Added comment in d62920820a3897c99237105e0b0961e6e17251e9